bin/MadGraph5_aMCatNLO/cards/production/13TeV/ZJets_LO_MLM/ZJetsToNuNu_HT-400to600/ZJetsToNuNu_HT400to600_run_card.dat

```diff
@@ -0,0 +1,255 @@
+# Number of helicities to sum per event (0 = all helicities)
+# 0 gives more stable result, but longer run time (needed for
+# long decay chains e.g.).
+# Use >=2 if most helicities contribute, e.g. pure QCD.
+#*************************************************************
+   0  = nhel          ! Number of helicities used per event
+#*******************                                                 
+# Standard Cuts
+#*******************                                                 
+#                                                                    
+#*********************************************************************
+# Minimum and maximum pt's (for max, -1 means no cut)                *
+#*********************************************************************
+<<<<<<< HEAD
+ 0.01  = ptj       ! minimum pt for the jets 
+=======
+ 10  = ptj       ! minimum pt for the jets 
+>>>>>>> f126f7f291d8749c0dd4fba078265712b1784185
+  0  = ptb       ! minimum pt for the b 
+ 10  = pta       ! minimum pt for the photons 
+ 0  = ptl       ! minimum pt for the charged leptons 
+  0  = misset    ! minimum missing Et (sum of neutrino's momenta)
+  0  = ptheavy   ! minimum pt for one heavy final state
+ 1.0 = ptonium   ! minimum pt for the quarkonium states
+ 1d5  = ptjmax    ! maximum pt for the jets
+ 1d5  = ptbmax    ! maximum pt for the b
+ 1d5  = ptamax    ! maximum pt for the photons
+ 1d5  = ptlmax    ! maximum pt for the charged leptons
+ 1d5  = missetmax ! maximum missing Et (sum of neutrino's momenta)
+#*********************************************************************
+# Minimum and maximum E's (in the lab frame)                         *
+#*********************************************************************
+  0  = ej     ! minimum E for the jets 
+  0  = eb     ! minimum E for the b 
+  0  = ea     ! minimum E for the photons 
+  0  = el     ! minimum E for the charged leptons 
+ 1d5   = ejmax ! maximum E for the jets
+ 1d5   = ebmax ! maximum E for the b
+ 1d5   = eamax ! maximum E for the photons
+ 1d5   = elmax ! maximum E for the charged leptons
+#*********************************************************************
+# Maximum and minimum absolute rapidity (for max, -1 means no cut)   *
+#*********************************************************************
+   5  = etaj    ! max rap for the jets 
+  1d2  = etab    ! max rap for the b
+ 2.5  = etaa    ! max rap for the photons 
+ 1d2  = etal    ! max rap for the charged leptons 
+ 0.6  = etaonium ! max rap for the quarkonium states
+   0  = etajmin ! min rap for the jets
+   0  = etabmin ! min rap for the b
+   0  = etaamin ! min rap for the photons
+   0  = etalmin ! main rap for the charged leptons
+#*********************************************************************
+# Minimum and maximum DeltaR distance                                *
+#*********************************************************************
+<<<<<<< HEAD
```

Comment:
I don’t know what is happening - This PR is from a fresh clean environment with a new brach… The previous PR was closed before I pushed the new one. What do you suggest ? In my area, I don’t see “conflicts"



> On 05 Mar 2015, at 20:53, Josh Bendavid <notifications@github.com> wrote:
> 
> In bin/MadGraph5_aMCatNLO/cards/production/13TeV/ZJets_LO_MLM/ZJetsToNuNu_HT-400to600/ZJetsToNuNu_HT400to600_run_card.dat <https://github.com/cms-sw/genproductions/pull/235#discussion_r25895753>:
> 
> > +#*********************************************************************
> > +# Maximum and minimum absolute rapidity (for max, -1 means no cut)   *
> > +#*********************************************************************
> > +   5  = etaj    ! max rap for the jets 
> > +  1d2  = etab    ! max rap for the b
> > + 2.5  = etaa    ! max rap for the photons 
> > + 1d2  = etal    ! max rap for the charged leptons 
> > + 0.6  = etaonium ! max rap for the quarkonium states
> > +   0  = etajmin ! min rap for the jets
> > +   0  = etabmin ! min rap for the b
> > +   0  = etaamin ! min rap for the photons
> > +   0  = etalmin ! main rap for the charged leptons
> > +#*********************************************************************
> > +# Minimum and maximum DeltaR distance                                *
> > +#*********************************************************************
> > +<<<<<<< HEAD
> and here...
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/cms-sw/genproductions/pull/235/files#r25895753>.
>